### PR TITLE
Update create-secure-guest-sharing-environment.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to state that SP and ODSC locations will be proactively blocked right after detection of sensitive information, irrespective of whether the document is shared or not.

Tested with a guest user who is a part of the team and it does block content that are blocked by DLP.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7652

File: microsoft-365/solutions/create-secure-guest-sharing-environment.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ To set up a guest access review
 13. Type a **Review name** and review the settings.
 14. Click **Create**.
 
-It's important to note that guests can be given access to teams or groups, or to individual files and folders. When given access to files and folders, guests may not be added to any particular group. If you want to do access reviews on guests who don't belong to a team or group, you can create a dynamic group in Azure AD to contain all guests and then create an access review for that group. Site owners can also manage [guest expiration for the site](https://support.microsoft.com/office/25bee24f-42ad-4ee8-8402-4186eed74dea)
+It's important to note that for SharePoint Online and OneDrive for Business locations documents will be proactively blocked right after detection of sensitive information, irrespective of whether the document is shared or not, for all external users, while internal users will continue to have access to the document.
```
Comment (JohanFreelancer9) [inline microsoft-365/solutions/create-secure-guest-sharing-environment.md]:
```suggestion
It's important to note that for SharePoint Online and OneDrive for Business locations, documents will be proactively blocked right after detection of sensitive information, irrespective of whether the document is shared or not, for all external users, while internal users will continue to have access to the document.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**